### PR TITLE
Http client

### PR DESCRIPTION
**Pull request by YulianaYershova**
Provided okhttp client

### CODE REVIEW

**Comment by ipt-cryptolabs** [comment]
Looks good, but what about tests? Are you going to add them to this branch or to a new one?

**Comment by YulianaYershova** [comment]
> Looks good, but what about tests? Are you going to add them to this branch or to a new one?

I'm going to add tests to new branch

File: http_client/src/main/java/com/fitgoal/BaseClient.java [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+import javax.inject.Inject;
+import java.io.IOException;
+import java.util.Arrays;
+
+import static java.net.HttpURLConnection.HTTP_BAD_REQUEST;
+
+abstract class BaseClient {
+
+    private static final MediaType JSON_MEDIA_TYPE = MediaType.parse("application/json; charset=utf-8");
+    private static final int UNPROCESSABLE_ENTITY_STATUS_CODE = 422;
```
Comment (ddovgal) [inline http_client/src/main/java/com/fitgoal/BaseClient.java]: Its OK, but fyi there is a class for all HTTP statuses called `org.eclipse.jetty.http.HttpStatus`
image

File: pom.xml [hunk 2]
```diff
@@ -14,6 +14,7 @@
         <module>service</module>
         <module>web</module>
         <module>dao</module>
+        <module>http_client</module>
```
Comment (ddovgal) [inline pom.xml]: Also, it's OK, just a little nit: for modules\artifacts `kebab-case` is preferable

Comment (YulianaYershova) [reply pom.xml]: Refactored

File: http_client/src/main/java/com/fitgoal/BaseClient.java (hunk 1 above)
Comment (YulianaYershova) [reply http_client/src/main/java/com/fitgoal/BaseClient.java]: Refactored. Thanks!

**Review by ipt-cryptolabs — APPROVED** [review]

**Pull request closed without merging**